db/migrate/20231228010428_create_responses.rb

```diff
@@ -0,0 +1,14 @@
+class CreateResponses < ActiveRecord::Migration[7.0]
+  def change
+    create_table :responses do |t|
+      t.references :submission, null: false, foreign_key: true
+      t.references :question, null: false, foreign_key: true
+      t.string :string_value
+      t.integer :integer_value
+      t.boolean :boolean_value
+      t.text :array_value, array: true
```

Comment:
Hmm I'am wondering if we want to store the value as a string and just serialize them? Maybe too much extra effort and it is best to just have these split up like this.

Reply:
Yeah - I think that would look a little cleaner, we'd have to transform it anyways, right? So store a value "true" and value_type "boolean"?

Reply:
So is the verdict that this is ok as-is? Part of me also thinks folding this into the `string_value` is fine but I pretty much hewed as closely as I could to Mae's design.

Reply:
Per mae - it sounds like the static types are good for calculations/sorting, which sounds handy so I think this is ok as-is